tests/test_bump_version.py

```diff
@@ -2,18 +2,20 @@
 """Some tests covering the bump_version code.
 """
 import os
-import tempfile
 import yaml
 
 import nf_core.bump_version
 import nf_core.create
 import nf_core.utils
 
 
-def test_bump_pipeline_version(datafiles):
+# pass tmp_path as argument, which is a pytest feature
+# see: https://docs.pytest.org/en/latest/how-to/tmp_path.html#the-tmp-path-fixture
+def test_bump_pipeline_version(datafiles, tmp_path):
```

Comment:
This type of tests are not a part of unittest: pytest fixtures have the `tmp_path` which is a temporary folder generated for each tests, and then deleted. Since it acts like context manager, I preferred to pass this argument to this type of tests.

Reply:
:100: